python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-pim-sparse-mode.j2

```diff
@@ -14,6 +14,8 @@
 BFD enabled: {{ router_pim_sparse_mode.ipv4.bfd | arista.avd.default(false) }}
 {%         if router_pim_sparse_mode.ipv4.rp_addresses is arista.avd.defined %}
 
+Make-before-break: {{ router_pim_sparse_mode.ipv4.make_before_break }}
+
```

Comment:
You need to move it above the existing `if` statement but then add your own `if block around the new printout, to only show it if the variable is defined. Make sure the blank line is between the `if` and the text, so no blank line between the text and the `endif`.

Reply:
got it, thanks Claus. I've added the if block now